kura/examples/org.eclipse.kura.example.ble.tisensortag.dbus/META-INF/MANIFEST.MF

```diff
@@ -1,12 +1,12 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
-Bundle-Name: org.eclipse.kura.example.ble.tisensortag.tinyb
-Bundle-SymbolicName: org.eclipse.kura.example.ble.tisensortag.tinyb;singleton:=true
+Bundle-Name: org.eclipse.kura.example.ble.tisensortag.dbus
+Bundle-SymbolicName: org.eclipse.kura.example.ble.tisensortag.dbus;singleton:=true
 Bundle-Version: 1.0.400.qualifier
```

Comment:
Since we have changed the name of the bundle, is it a new one? If so, have we to start again from 1.0.0?

Reply:
Good question. It's technically the same bundle but I figured calling it "tinyb" after a library no longer in use didn't make much sense.

I will let you instruct as to your preference on versioning.